src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -63,16 +78,13 @@ class TeamMembers extends React.Component {
     }
   }
 
-  debouncedFetchMembersRequest = debounce(function (query) {
-    this.setState(
-      {
-        dropdownBusy: true,
-      },
-      () => this.fetchMembersRequest(query)
-    );
-  }, 200);
+  debouncedFetchMembersRequest = debounce(
+    (query: string) =>
+      this.setState({dropdownBusy: true}, () => this.fetchMembersRequest(query)),
+    200
+  );
 
-  removeMember(member) {
+  removeMember(member: Member) {
```

Comment:
maybe you could write as below:

```suggestion
   removeMember = (member: Member) => () => {
```

Reply:
Either way it creates a new function every render, I prefer to just keep it at the call site, since that pattern is more common.